code/game/objects/items/devices/radio/beacon.dm

```diff
@@ -8,7 +8,7 @@
 /obj/item/device/radio/beacon/Initialize()
 	. = ..()
 	GLOB.radio_beacon_list += src
-	flags_atom &= ~USES_HEARING // This subtype apparenlty does not want to hear stuff
+	LAZYREMOVE(GLOB.hearing_objects, src)
```

Comment:
Unless Initialize can occur again for the same object, it shouldn't ever already be in the list to need to be removed yet.